### PR TITLE
Transfer tokens, stake BALN & unstake sICX contract calls

### PR DESCRIPTION
**Pull request by tuan-tl**
- [x] Transfer sICX, BALN, bnUSD
- [x] Stake BALN
- [x] Unstake sICX (need more testing)

### CODE REVIEW

**Comment by BennyOptions** [comment]
Transfer sICX = success
https://bicon.tracker.solidwallet.io/transaction/0x07785b52dea298e3a68e347d84a8955ea9d6a902c7bc9c36e56b2072782e58c4

Send bnUSD = N/A (UI Issue)
- UI does not allow the user to type @WeiLi512 

Stake BALN = N/A (UI Issue)
- Could not verify, I have 0 BALN and the UI does not allow typing @WeiLi512

File: src/packages/BalancedJs/contracts/Baln.ts [hunk 1]
```diff
@@ -56,4 +56,29 @@ export default class Baln extends Contract {
     console.log(payload);
     return this.callIconex(payload);
   }
+
+  public async transfer(toAddress: string, value: number): Promise<any> {
```
Comment (WeiLi512) [inline src/packages/BalancedJs/contracts/Baln.ts:60]: I'd like to use the names of the same parameters as names of smart contract method params. 
toAddress -> to

File: src/packages/BalancedJs/contracts/Baln.ts [hunk 2]
```diff
@@ -56,4 +56,29 @@ export default class Baln extends Contract {
...
+        _to: toAddress,
+        _value: '0x' + IconAmount.of(value, IconAmount.Unit.ICX).toLoop().toString(16),
+      },
+      value: 0,
+    });
+
+    return this.callIconex(callParams);
+  }
+
+  async stakeBALN(value: number): Promise<ResponseJsonRPCPayload> {
```
Comment (WeiLi512) [inline src/packages/BalancedJs/contracts/Baln.ts:73]: I'd like to use the names of the same method as the names of the smart contract method.

Comment (WeiLi512) [reply src/packages/BalancedJs/contracts/Baln.ts:73]: stakedBALN -> stake

File: src/packages/BalancedJs/contracts/bnUSD.ts [hunk 3]
```diff
@@ -72,4 +72,17 @@ export default class bnUSD extends Contract {
     });
     return this.callIconex(payload);
   }
+
+  public async transfer(toAddress: string, value: number): Promise<any> {
```
Comment (WeiLi512) [inline src/packages/BalancedJs/contracts/bnUSD.ts:76]: I'd like to use the names of the same parameters as names of smart contract method params.
toAddress -> to

File: src/packages/BalancedJs/contracts/sICX.ts [hunk 4]
```diff
@@ -103,4 +103,31 @@ export default class sICX extends Contract {
 
     return this.call(callParams);
   }
+
+  public async transfer(toAddress: string, value: number): Promise<any> {
```
Comment (WeiLi512) [inline src/packages/BalancedJs/contracts/sICX.ts:107]: I'd like to use the names of the same parameters as names of smart contract method params.
toAddress -> to

File: src/packages/BalancedJs/contracts/sICX.ts [hunk 5]
```diff
@@ -103,4 +103,31 @@ export default class sICX extends Contract {
...
+        _to: toAddress,
+        _value: '0x' + IconAmount.of(value, IconAmount.Unit.ICX).toLoop().toString(16),
+      },
+      value: 0,
+    });
+
+    return this.callIconex(callParams);
+  }
+
+  async unstakeSICX(value: number): Promise<ResponseJsonRPCPayload> {
```
Comment (WeiLi512) [inline src/packages/BalancedJs/contracts/sICX.ts:120]: name convention

**Review by WeiLi512 — CHANGES_REQUESTED** [review]

**Review by WeiLi512 — APPROVED** [review]

**Pull request merged**